yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnAllocator.scala

```diff
@@ -353,7 +353,7 @@ private[yarn] class YarnAllocator(
 
     } else if (missing < 0) {
       val numToCancel = math.min(numPendingAllocate, -missing)
-      logInfo(s"Canceling requests for $numToCancel executor containers")
+      logInfo(s"Canceled requests for $numToCancel executor container(s)")
```

Comment:
We can show the old value of course but it doesn't solve my problem.

16/06/07 18:53:48 INFO yarn.YarnAllocator: Driver requested a total number of 614 executor(s).
16/06/07 18:53:48 INFO yarn.YarnAllocator: Canceling requests for 5 executor containers

It says Driver requested 614 executors (so it supposed to be already happened). But then it says Canceling 5 executors (it is happening now). This is not correct. The 614 executor is less with 5 executors then the previous status update (619).

The root cause is that these two messages are swapped, this would make total sense:
16/06/07 18:53:48 INFO yarn.YarnAllocator: Canceling requests for 5 executor containers
16/06/07 18:53:48 INFO yarn.YarnAllocator: Driver requested a total number of 614 executor(s).

The minimum change to fix this is the one introduced in this pull request, just to reword the Cancel message.